api/go.mod

```diff
@@ -63,27 +185,31 @@ replace (
 	github.com/gojek/turing/engines/experiment => ../engines/experiment
 	github.com/gojek/turing/engines/router => ../engines/router
 
-	k8s.io/api => k8s.io/api v0.16.4
+	k8s.io/api => k8s.io/api v0.22.7
 
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.16.4
-	k8s.io/apimachinery => k8s.io/apimachinery v0.16.4
-	k8s.io/apiserver => k8s.io/apiserver v0.16.4
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.16.4
-	k8s.io/client-go => k8s.io/client-go v0.16.4
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.16.6
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.16.6
-	k8s.io/code-generator => k8s.io/code-generator v0.16.6
-	k8s.io/component-base => k8s.io/component-base v0.16.6
-	k8s.io/cri-api => k8s.io/cri-api v0.16.6
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.16.6
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.16.6
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.16.6
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.16.6
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.16.6
-	k8s.io/kubectl => k8s.io/kubectl v0.16.4
-	k8s.io/kubelet => k8s.io/kubelet v0.16.6
-	k8s.io/kubernetes => k8s.io/kubernetes v1.16.4
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.16.6
-	k8s.io/metrics => k8s.io/metrics v0.16.6
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.16.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.22.7
```

Comment:
Do we still need these replacements after the upgrade of all of the dependencies?

Reply:
Yes, these 0.16.6 corresponds to k8s 1.16, so 0.22 corresponds to 1.22. But if you are asking about the aliasing then yes it is needed still.